package.json

```diff
@@ -26,18 +26,19 @@
   "scripts": {
     "build": "tsc",
     "prepare": "npm run build",
-    "test": "jest"
+    "test": "jest --transformIgnorePatterns 'node_modules/(?!my-library-dir)/'"
```

Comment:
what does this do?

Reply:
This is resolved since I am using a different query string package